### PR TITLE
Improve stdout exporter

### PR DESCRIPTION
**Pull request by RayZhao1998**
This is the PR for #11.

- [x] JSON output
- [x] debug switch

Don't know where to implement the `debug switch`. Waiting for discussion.

## UPDATE

`debug switch` has been added in constructor of StdoutExporter.

### CODE REVIEW

File: Sources/Exporters/Stdout/StdoutExporter.swift [hunk 1]
```diff
@@ -41,3 +51,29 @@ public class StdoutExporter: SpanExporter {
...
+    private let attributes: [String: AttributeValue]
+    
+    init(span: SpanData) {
+        self.span = span.name
+        self.traceId = span.traceId.hexString
+        self.spanId = span.spanId.hexString
+        self.spanKind = span.kind.rawValue
+        self.traceFlags = span.traceFlags
+        self.traceState = span.traceState
+        self.parentSpanId = span.parentSpanId?.hexString ?? "no Parent"
```
Comment (nacho-undefinedlabs) [inline Sources/Exporters/Stdout/StdoutExporter.swift]:
```suggestion
        self.parentSpanId = span.parentSpanId?.hexString ?? SpanId.invalid.hexString
```

**Comment by nacho-undefinedlabs** [comment]
And about the debug switch, maybe a parameter in the constructor with a default value, and using pretty output option  for the JSON could be enough, what do you think?

File: Sources/Exporters/Stdout/StdoutExporter.swift (hunk 1 above)
Comment (RayZhao1998) [reply Sources/Exporters/Stdout/StdoutExporter.swift]: Fixed. I just copied the previous output before.

**Comment by RayZhao1998** [comment]
`debug switch` has been added. Please review again. @nacho-undefinedlabs

**Comment by RayZhao1998** [comment]
I signed it

**Comment by RayZhao1998** [comment]
I signed it

**Comment by nacho-undefinedlabs** [comment]
PR merged, thanks for your contribution

**Comment by RayZhao1998** [comment]
Thanks! :)

Any other work I can do? I'm working on the `Server Distributed Tracing in Swift` as my Google Summer of Code project. So I hope I can have more contributions on `OpenTelemetry-Swift`.

**Comment by nacho-undefinedlabs** [comment]
Of course there is!! Just let me create GitHub issues for some of the needs the project has right now, so you can pick up something to work on.

Thanks

**Pull request merged**